Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.java

```diff
@@ -924,6 +1024,18 @@ private static InputOverlayDrawableButton initializeOverlayButton(Context contex
       case ButtonType.CLASSIC_BUTTON_ZR:
         scale = 0.25f;
         break;
+      case ButtonType.DRUMS_PAD_BASS:
+        scale = 0.425f;
+        break;
+      case ButtonType.DRUMS_PAD_GREEN:
+      case ButtonType.DRUMS_PAD_RED:
+        scale = 0.4f;
+        break;
+      case ButtonType.DRUMS_PAD_BLUE:
```

Comment:
Why is the blue pad smaller than the green and red pads? Those three are identical on a physical controller.

Reply:
As far as I tried. it's easier that way, than following the original wii drum kit's appearance

Reply:
I think making all pads the smaller size and tweaking their position, would look fine.